code/game/machinery/doors/airlock.dm

```diff
@@ -1338,6 +1338,9 @@
 	if(!hasPower())
 		to_chat(user, "<span class='warning'>The cryptographic sequencer connects to \the [src]'s ID scanner, but nothing happens.</span>")
 		return FALSE
+	if(AIRLOCK_SECURITY_NONE)
```

Comment:
```suggestion
	if(security_level != AIRLOCK_SECURITY_NONE)
```
It should be like this, AIRLOCK_SECURITY_NONE is a define, not a variable on the airlock.

Reply:
Aah! I understand now, I figured it looked like a Java variable. Thank you!